orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -48,13 +48,6 @@
 /**
  * OrientDB client for YCSB framework.
  * 
- * Properties to set:
- * 
- * orientdb.url=local:C:/temp/databases or remote:localhost:2424 <br>
- * orientdb.database=ycsb <br>
- * orientdb.user=admin <br>
- * orientdb.password=admin <br>
- * 
  * @author Luca Garulli
```

Comment:
For my own knowledge, should author tags not exist at all? I think I am guilty of auto generated author tags as well.